src/main/java/org/elasticsearch/river/twitter/TwitterRiver.java

```diff
@@ -62,6 +62,8 @@
     private final String oauthAccessToken;
     private final String oauthAccessTokenSecret;
 
+    private final int retryAfter;
```

Comment:
Better using `private final TimeValue retryAfter;`

Then you can set it using:

```java
this.retryAfter = XContentMapValues.nodeTimeValue(twitterSettings.get("retry_after"), TimeValue.timeValueSeconds(1));

// Or if it does not work something like

this.retryAfter = TimeValue.parseTimeValue(XContentMapValues.nodeStringValue(
                    twitterSettings.get("retry_after"), "1s"), TimeValue.timeValueSeconds(1));
```